modulus.py

```diff
@@ -8,13 +8,11 @@
 
 if (a > 0 and b > 0) or (a < 0 and b < 0):
     hasil = a - int(a/b) * b
-elif a == 0 and b > 0:
+elif (a == 0 and b > 0) or (a == 0 and b == 0):
```

Comment:
pertama aku baru tau kalo 0 mod 0 hasilnya 0, aku cari soalnya error
nggak usah buat operator baru sebenernya lagi
a == 0 and b >= 0 udah cukup
btw makasi yaw wkwkwk
aku make text editor jadinya nggak keliatan kalo berapa space yang bener
jadinya cuman seingetku deh